packages/pigeon/platform_tests/test_plugin/example/ios/Runner/AppDelegate.swift

```diff
@@ -5,7 +5,7 @@
 import Flutter
 import UIKit
 
-@UIApplicationMain
+@main
```

Comment:
Unrelated to the extension error. The Flutter tool updated this when I built the example.